### ISSUE TITLE
xfce4-terminal floating window will become larger and larger 

### ISSUE THREAD

**Issue by tumashu** [ch11ng/exwm#140]

**Comment by ch11ng** [comment]
This is because resizing an Emacs frame is not accurate currently.  I hope we'll find a way to fix it.

P.S.  Please don't open issues without mentioning how to reproduce them.  It's simply confusing.

**Comment by tumashu** [comment]
```
P.S. Please don't open issues without mentioning how to reproduce them. It's simply confusing.
```
1. Open xfce4-terminal
2. toggle it to floating window mode
3. you will see it getting larger and larger

Do you mean this?

**Comment by ch11ng** [comment]
Fixed in c8c0bc7b60c043d47e153e40c2ae139f513e9358.  I was not able to reproduce this bug until it happened to me today.  So please provide log next time when you report a bug such that I can fix it timely.

**Comment by tumashu** [comment]
```
please provide log next time when you report a bug such that I can fix it timely.
```
OK
